examples/terrain/main.rs

```diff
@@ -151,10 +118,6 @@ fn main() {
     let mut glfw = glfw::init(glfw::FAIL_ON_ERRORS)
         .ok().expect("Failed to initialize glfs-rs");
 
-    glfw.window_hint(glfw::WindowHint::ContextVersion(3, 2));
```

Comment:
@bjz I recall there was at least one platform that behaved strangely if you did not provide these. Do you remember?